public/css/nexthome.css

```diff
@@ -1,530 +1,649 @@
+    font-size: 16px;
+}
+#mainContainer.stats.collaspe .statCardWrap .content .data .footing {
+    display: none;
+}
+#mainContainer.stats.collaspe .statCardWrap .content .data .progress + .footing {
+    display: block;
+    position: absolute;
+    color: #ffffff;
+    top: 24px;
+    left: calc(5% + 7px);
+    z-index: 99;
+}
 .img-size {
-  height: 20px;
+    height: 20px;
 }
-
 .networkStatsWrap,
 .blockRewardsWrapper,
 .proofofstakeWrapper {
-  padding-right: 50px; }
+    padding-right: 50px;
+}
 .info-block .expanded-block .row .col-12 img {
     margin-right: 3%;
     margin-top: 2%;
-    height: 20px; }
+    height: 20px;
+}
 @media (min-width: 1400px) {
-  .container {
-    max-width: 1280px !important; }
-    .container .logo a img {
-      width:100px !important; } }
+    .container {
+        max-width: 1280px !important;
+    }    .container .logo a img {
+        width: 100px !important;
+    }
+}
 @media (min-width: 800px) {
-  header .right .searchbox.full input {
-    flex: 1; }}
+    header .right .searchbox.full input {
+        flex: 1;
+    }
+}
 @media (max-width: 600px) {
-  header .right .searchbox input {
-    width: 150px; }}
-
-@media only screen and (max-width: 768px)  {
-  .container {
-    max-width: 85% !important;
-  }
-  .container .logo a img {
-    width:70px !important;
-  }
-  .col-md-6 {
-    flex: 0 0 50%;
-    max-width: 100%;
-  }
-  .col-sm-4 {
-    flex: 0 0 33.333333%;
-    max-width: 33.333333%;
-  }
-  .col-lg-4 {
-    flex: 0 0 100%;
-    max-width: 33.333333%;
-  }
-  .networkStatsWrapper .statCard {
-    margin-right: 1% !important;
-    min-height: fit-content;
-  }
-  .dcr-icon {
-    width: 100px;
-  }
-  .dcr-icon-container {
-    width: 100px;
-  }
-}
-
-@media only screen and (max-width: 576px)  {
-  .networkStatsWrapper {
-    padding: 40px 0px;
-  }
-  .decBlockchainWrapper {
-    padding: 0px 0px;
-  }
-  .col-lg-4 {
-    flex: 0 0 100%;
-    max-width: 50%;
-  }
-  .col-sm-4 {
-    flex: 0 0 100%;
-    max-width: 100%;
-    padding-right: 0px;
-    padding-left: 0px;
-  }
-  .dcr-icon {
-    width: 60px;
-  }
-  .dcr-icon-container {
-    width: 60px;
-  }
-  .navbar-fixed-bottom{
-    font-size: 12px;
-  }
+    header .right .searchbox input {
+        width: 150px;
+    }
 }
+@media only screen and (max-width: 768px) {
+    .container {
+        max-width: 85% !important;
+    }    .container .logo a img {
+        width: 70px !important;
+    }    .col-md-6 {
+        flex: 0 0 50%;
+        max-width: 100%;
+    }    .col-sm-4 {
+        flex: 0 0 33.333333%;
+        max-width: 33.333333%;
+    }    .col-lg-4 {
+        flex: 0 0 100%;
+        max-width: 33.333333%;
+    }    .networkStatsWrapper .statCard {
+        margin-right: 1% !important;
+        min-height: fit-content;
+    }    .dcr-icon {
+        width: 100px;
+    }    .dcr-icon-container {
+        width: 100px;
+    }
+}
+@media only screen and (max-width: 576px) {
+    .decBlockchainWrapper {
```

Comment:
This formatting is totally non-standard. 
Please just format it like this
https://css-tricks.com/different-ways-to-format-css/#article-header-id-0
Since that's by far the most common.